openedx/core/djangoapps/course_live/admin.py

```diff
@@ -0,0 +1,35 @@
+"""
+Admin interface for course live configuration
+"""
+from django.contrib import admin
+from simple_history.admin import SimpleHistoryAdmin
+
+from openedx.core.djangoapps.course_live.forms import CourseLiveConfigurationForm
+from openedx.core.djangoapps.course_live.models import CourseLiveConfiguration
+
+
+class CourseLiveConfigurationAdmin(SimpleHistoryAdmin):
+    """
+    Customize the admin interface for the course live configuration
+    """
+    form = CourseLiveConfigurationForm
+
+    fieldsets = (
+        (None, {
+            'fields': ('course_key', 'enabled', 'lti_configuration', 'pii_share_username', 'pii_share_email',
```

Comment:
We do not need to override admin for this to add pii. It will be configured via course authoring we we will need to add pii there.